bugalink-frontend/pages/search/result.tsx

```diff
@@ -317,33 +317,33 @@ export default function SearchResults() {
       <div className="divide-y-2 divide-light-gray">
         {isLoading || isError
           ? [1, 2, 3, 4, 5].map((i) => (
-              <TripCardSkeleton
-                key={i}
-                className="rounded-md bg-white outline outline-1 outline-light-gray"
-              />
-            ))
+            <TripCardSkeleton
+              key={i}
+              className="rounded-md bg-white outline outline-1 outline-light-gray"
+            />
+          ))
           : searchResults.map((trip) => (
-              <Link
+            <Link
+              key={trip.name}
+              href="/ride/V1StGXR8_Z5jdHi6B-myT/details?requested=false"
+              className="w-full"
+            >
+              <TripCard
                 key={trip.name}
-                href="/ride/V1StGXR8_Z5jdHi6B-myT/detailsOne?requested=false"
-                className="w-full"
-              >
-                <TripCard
-                  key={trip.name}
-                  type={trip.type}
-                  rating={trip.rating}
-                  name={trip.name}
-                  gender={trip.gender}
-                  avatar={trip.avatar}
-                  origin={trip.origin}
-                  destination={trip.destination}
-                  date={trip.date}
-                  price={trip.price}
-                  className="rounded-md bg-white outline outline-1 outline-light-gray"
-                  href={NEXT_ROUTES.RIDE_DETAILS_ONE(trip.id)}
-                />
-              </Link>
-            ))}
+                type={trip.type}
+                rating={trip.rating}
+                name={trip.name}
+                gender={trip.gender}
+                avatar={trip.avatar}
+                origin={trip.origin}
+                destination={trip.destination}
+                date={trip.date}
+                price={trip.price}
+                className="rounded-md bg-white outline outline-1 outline-light-gray"
+                href={NEXT_ROUTES.RIDE_DETAILS_ONE(trip.id)}
```

Comment:
`RIDE_DETAILS_ONE` should be renamed to `RIDE_DETAILS` now that we don't have `detailsTwo`

Reply:
Refactor done